src/Components/Aspire.Azure.Search.Documents/ConfigurationSchema.json

```diff
@@ -0,0 +1,132 @@
+{
+  "definitions": {
+    "logLevel": {
+      "properties": {
+        "Azure": {
+          "$ref": "#/definitions/logLevelThreshold"
+        },
+        "Azure.Core": {
+          "$ref": "#/definitions/logLevelThreshold"
+        },
+        "Azure.Identity": {
+          "$ref": "#/definitions/logLevelThreshold"
+        }
+      }
+    }
+  },
+  "properties": {
+    "Aspire": {
+      "type": "object",
+      "properties": {
+        "Azure": {
+          "type": "object",
+          "properties": {
+            "Search": {
+              "type": "object",
+              "properties": {
+                "Documents": {
+                  "type": "object",
+                  "properties": {
+                    "ClientOptions": {
```

Comment:
Looks like we're missing the [`Audience`](https://learn.microsoft.com/en-us/dotnet/api/azure.search.documents.searchclientoptions.audience?view=azure-dotnet#azure-search-documents-searchclientoptions-audience).
@eerhardt - is there any chance we could update your config generator to turn our [extensible enum pattern](https://azure.github.io/azure-sdk/dotnet_implementation.html#dotnet-enums) into strings here?

Reply:
Let's chat about that offline before filing - I'm not sure it's something you'd want to bake into the standard config extensions.  It's our unique pattern to work around https://github.com/dotnet/csharplang/issues/2849 and doesn't have any clear tagging attributes/discovery mechanism.  We could add those in a future release, but ultimately the Azure config story probably won't be ConfigurationBinder friendly anyway for a lot of reasons (extensible enums, polymorphic auth options, compat with existing untyped Microsoft.Extensions.Azure behavior, etc.).